src/docs/forms/components/CalendarInput/examples/desc/04-iconCLass.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export default class iconClass extends React.Component {
```

Comment:
Always upper-case first letter of classes. Rename your file accordingly. Also, L in iconCLass is upper-cased so change it too.

Reply:
Partially resolved as of 1103 30 July. Rename file correspondingly.